src/app/new-page/component/content-flow-item/content-flow-item.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import {
+  Content,
+  FlowItem,
+  FlowWatcher
+} from 'src/app/services/xml-parser-service/xmp-parser.service';
+import { PageService } from '../../service/page-service.service';
+
+@Component({
+  selector: 'app-content-new-flow-item',
+  templateUrl: './content-flow-item.component.html',
+  styleUrls: ['./content-flow-item.component.css']
+})
+export class ContentFlowItemNewComponent implements OnChanges {
+  // eslint-disable-next-line @angular-eslint/no-input-rename
+  @Input() item: FlowItem;
+  contents: Content[];
+  ready: boolean;
+  state: any;
+  isHidden: boolean;
+  flowWatcher: FlowWatcher;
```

Comment:
I would name this `isHiddenWatcher`, that way if in the future you start watching an additional data stream you can track the multiple active data streams and cancel all of them when appropriate